server.py

```
@@ -28,6 +30,6 @@ async def login(request: Request):
     return templates.TemplateResponse("auth/login.html", {"request": request})
 
 
-@app.get("auth/register")
+@app.get("/auth/register")
 async def index(request: Request):
```

Ruff (0.8.2)

Redefinition of unused `index` from line 24

(F811)